release_planning/pupet_module.rb

```diff
@@ -0,0 +1,12 @@
+
+class PuppetModule
```

Comment:
this is not used by anything outside of the release scripts. it should just live in the release planning script